airtime_mvc/application/views/scripts/playlist/update.phtml

```diff
@@ -2,6 +2,28 @@
 $items = $this->contents;
 $isSmartBlock = ($this->obj instanceof Application_Model_Block);
 $isPlaylist = ($this->obj instanceof Application_Model_Playlist);
+if ($this->poolCount) { ?>
+    <div class='sp_text_font sp_text_font_bold'>
+        <span id='sp_pool_count' class='sp_text_font sp_text_font_bold'>
+        <?php
+        if ($this->poolCount > 0) {
+            echo $this->poolCount;
+            echo ngettext(" file meets the criteria", " files meets the criteria", $this->poolCount);
```

Comment:
So are you suggesting we basically just take away the if poolcount > 0 and just display files meet the criteria and see if ngettext handles the 0 case correctly ? I'll test that out.

Reply:
Ok, so I changed this and it works. I needed to also use showPoolCount at the top vs. poolCount because it made it so the 0 case never displayed and also without that it was displaying 0 files matched when a new form was generated.

I tested it and this resolves both issues. I needed to keep one 0 case check because I wanted to show the ! if no files matched. 

I also found some dead code that was not being used at all and deleted it.